### PR TITLE
Home Owners UXA feedback

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* /bcgov/entity#12452

*Description of changes:*

- Home Owners page UXA feedback (mostly styling)

Screen Shot 2022-08-04 at 1 38 43 PM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by doug-lovett — APPROVED** [review]
Looks great, thanks for the screenshot.

File: ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue [hunk 1]
```diff
@@ -123,9 +131,10 @@ export default defineComponent({
 
     const localState = reactive({
       currentlyEditingHomeOwnerId: -1,
-      isEditing: computed((): boolean => {
-        return localState.currentlyEditingHomeOwnerId >= 0
-      })
+      isEditingMode: computed(
+        (): boolean => localState.currentlyEditingHomeOwnerId >= 0
+      ),
```
Comment (cameron-eyds) [inline ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue:136]: I know it's all the same but perhaps we preserve the formatting of the computed values the same as done elsewhere.

**Review by cameron-eyds — APPROVED** [review]
Looks good, just the 1 comment 👍

File: ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue (hunk 1 above)
Comment (dimak1) [reply ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue:136]: It's the prettier that formats it like this...

Comment (cameron-eyds) [reply ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue:136]: ahh fair enough :) 
100% All the same for sure, just nice to follow the previously done stuff for consistency

**Pull request merged**